### ISSUE TITLE
Accuracy of M3SDA on DomainNet

### ISSUE THREAD

**Issue by omg777** [KaiyangZhou/Dassl.pytorch#17]
Hi, thanks for sharing the nice codes.

I have some trouble to get accuracy of M3SDA on DomainNet.
With below command, I got 0.55% of accuracy (error:99.45%).
What`s wrong with this ?
```
python tools/train.py --root /database --trainer M3SDA --source -domains clipart --target-domains infograph --dataset-config-file configs/datasets/da/domainnet.yaml --config-file configs/trainers /da/m3sda/domainnet.yaml --output-dir output/M3SDA_CI_DOMAINNET 
```

**Comment by KaiyangZhou** [comment]
can you show the log file please?

**Comment by omg777** [comment]
@KaiyangZhou 
I ran it again and it worked fine.
Thanks.

By the way, in your semi-supervised learning (FixMatch) code, the cifar-10 data split provides [train_u: 46,000, train_x: 4,000].
But, in the paper it seems that train_u should be 41,000, can you confirm this?

**Comment by KaiyangZhou** [comment]
please check this paper "Realistic Evaluation of Deep Semi-Supervised Learning Algorithms", and modify the dataset config if necessary

please refer to this code as well https://github.com/KaiyangZhou/Dassl.pytorch/blob/master/dassl/data/datasets/ssl/cifar.py#L12
